tests/mock_tables/config_db.json

```diff
@@ -498,6 +498,7 @@
     },
     "VLAN|Vlan1000": {
         "dhcp_servers@": "192.0.0.1,192.0.0.2,192.0.0.3,192.0.0.4",
+        "dhcpv6_servers@": "fc02:2000::1,fc02:2000::2",
```

Comment:
Shouldn't this be a list 
        "dhcpv6_servers": ["fc02:2000::1","fc02:2000::2"],